pkg/services/ngalert/state/manager.go

```diff
@@ -265,3 +267,22 @@ func (st *Manager) createAlertAnnotation(new eval.State, alertRule *ngModels.Ale
 		return
 	}
 }
+
+func (st *Manager) staleResultsHandler(alertRule *ngModels.AlertRule, states map[string]*State) {
+	allStates := st.GetStatesForRuleUID(alertRule.OrgID, alertRule.UID)
+	for _, s := range allStates {
+		_, ok := states[s.CacheId]
+		if !ok && isItStale(s.LastEvaluationTime, time.Duration(alertRule.IntervalSeconds)) {
```

Comment:
```suggestion
		if !ok && isItStale(s.LastEvaluationTime, alertRule.IntervalSeconds * time.Second) {
```
I think that this would be clearer than having to multiply with `time.Second` [here](https://github.com/grafana/grafana/blob/299f79d6c8fc5efc073cf4026ebbb75918e67827/pkg/services/ngalert/state/manager.go#L287)

Reply:
Makes sense. I varied it slightly but I think it preserves the intention of your comment.